packages/webapp/src/utils/developer-constants.js

```diff
@@ -127,46 +132,12 @@ const ROUTING_REGISTER_OPTIONS = {
   RECORD: 'record-off-site'
 }
 
+const views = Object.fromEntries(
```

Comment:
This isn't needed as the final route and views are created and exported in `constants.js` and we don't use `developer-constants.js` directly

Reply:
done